### DISCUSSION TITLE
Convert track local time to local time

### DISCUSSION THREAD

**Discussion by BeunHaas34** [f1multiviewer/issue-tracker#55] [Feature requests] [answered]
![image](https://user-images.githubusercontent.com/4223026/174456453-97793ae0-a168-40a5-8518-1f5a7cfb0ed0.png)

**Comment by nikitaschaffner** [comment]
Saw this and agree. Maybe add both local time and current time. 
In addition to this, if it's possible to connect those two, I would love to see the current local time on the live timing screen as well.

**Comment by f1multiviewer** [comment, marked as answer]
Done! You can hover to see your own local time!
